tests/candidateApp-advanced.spec.ts

```diff
@@ -1,33 +1,59 @@
 import { test, expect } from "@playwright/test";
 import { faker } from "@faker-js/faker";
 import path from "path";
+import { TRANSLATIONS as T } from './utils/translations';
 import { Route } from "../frontend/src/lib/utils/navigation/route";
-import candidateAppTranslationsEn from "../frontend/src/lib/i18n/translations/en/candidateApp.json";
-import candidateAppTranslationsFi from "../frontend/src/lib/i18n/translations/fi/candidateApp.json";
-import ariaTranslations from "../frontend/src/lib/i18n/translations/en/aria.json";
-import questionsTranslations from "../frontend/src/lib/i18n/translations/en/questions.json";
+import mockInfoQuestions from "../backend/vaa-strapi/src/functions/mockData/mockInfoQuestions.json";
+import mockQuestionTypes from "../backend/vaa-strapi/src/functions/mockData/mockQuestionTypes.json";
```

Comment:
The question types contain the labels for enumerated questions which in turn are needed for locating elements. Thus, we also import them instead of hard-coding labels or using translation terms (which are mostly deprecated anyway).